README.md

```diff
@@ -5,38 +5,99 @@
 
 This project is a modern implementation of the Ubuntu Desktop installer, using [subiquity](https://github.com/canonical/subiquity) as a backend and [Flutter](https://flutter.dev/) for the UI.
 
-## Checkout & Run
+![Preview Screenshot](.github/docs/images/preview_screenshot.png)
 
-Grab the source:
+## Availability
+
+The new Ubuntu Desktop installer is *planed* to ship with the next Ubuntu Desktop LTS 22.04 Impish Idri.
+
+## Test the new Ubuntu Desktop installer
+
+If you want to test the new installer before 22.04 goes live, you can download a preview canary image below and install it for example in a virtual machine:
+
+[Download](http://cdimage.ubuntu.com/impish/daily-canary/current/)
+
+
+## Building and running manually
+
+To build the installer manually you need to have the [Flutter UI toolkit](https://flutter.dev) installed and the source code in this repository downloaded on your computer.
+
+To install Flutter you have two options: a manual installation by cloning the flutter repo and adding it tou your $PATH or installing the Flutter snap.
+
+### Either: Install Flutter manually
+
+Install the Flutter dependencies:
 ```sh
-git clone https://github.com/canonical/ubuntu-desktop-installer.git
-cd ubuntu-desktop-installer
-git submodule update --init --recursive
+sudo apt install git curl cmake meson clang cmake pkg-config libgtk-3-dev
 ```
-
-Install and configure Flutter:
+Install Flutter:
 ```sh
-snap install flutter --classic
+mkdir ~/development
+cd ~/development
+git clone https://github.com/flutter/flutter.git -b master
+echo 'export PATH="$PATH:$HOME/development/flutter/bin"' >> ~/.bashrc
+```
+
+### Or: Install Flutter with Snapd
+
+```
+sudo snap install flutter --classic
+```
+
+### Configure Flutter
+
+In any case you need to activate the linux desktop support in the flutter SDK.
+To do so, run the following commands:
+
+```
 flutter upgrade
 flutter config --enable-linux-desktop
 ```
 
-Install dependencies:
+### Download and build the Ubuntu Desktop installer
+
+Grab the source:
+```sh
+git clone https://github.com/canonical/ubuntu-desktop-installer.git
+cd ubuntu-desktop-installer
+git submodule update --init --recursive
+```
+
+Install the Python (subitquity), Dart and Flutter dependencies:
 ```sh
 cd packages/subiquity_client
 dart pub get
 cd subiquity
 make install_deps
-cd ../../ubuntu_desktop_installer
+cd ../../udev/
+flutter pub get
+cd ../ubuntu_wsl_setup/
+flutter pub get
+cd ../ubuntu_wizard/
+flutter pub get
+cd ../logger/
+flutter pub get
```

Comment:
there's no need to `cd` through all package dirs. even `flutter pub get` in `packages/ubuntu_desktop_installer` is optional because `flutter run` does it automatically.

Reply:
It was needed to resolve all dependency issues in vscode at least.

Reply:
Doesn't the Dart-Code extension show a popup that offers running pub get for all packages? As for the command-line, we are using [Melos](https://melos.invertase.dev/) to avoid having to `cd` and `pub get` back and forth. :)

These instructions here can be a minimal set of commands to get the app running. ​The goal is not to scare anyone away. :) There are more elaborate instructions and Melos examples in [CONTRIBUTING.md](https://github.com/canonical/ubuntu-desktop-installer/blob/main/CONTRIBUTING.md#melos)